spk/mediainfo/Makefile

```diff
@@ -0,0 +1,35 @@
+SPK_NAME = mediainfo
+SPK_VERS = 0.7.72
+SPK_REV = 1
+SPK_ICON = src/mediainfo.png
+
+DEPENDS  = cross/$(SPK_NAME)
+
+RELOAD_UI = yes
+STARTABLE = no
+
+MAINTAINER = cytec
```

Comment:
For now we use SynoCommunity to avoid user confusion. There is a maintainer branch on which we put SynoCommunity/maintainer but it's too long so we'd need something else.

Let's put SynoCommunity there for now and we'll put the maintainer once I've implemented a developer page on spkrepo.